client_app/src/components/shelter/EditStatusConfirmationModal.js

```diff
@@ -19,13 +21,19 @@ export const EditStatusConfirmationModal = props => {
   const renderData = () => {
     return (
       <div>
-        <p> Do you want to {!shift.status ? "open" : "close"} volunteer shift registration on {shift.date} from {shift.startTime} to {shift.endTime}? </p>
-        <button onClick={() => onYesClick()}>
-          Yes
```

Comment:
When run, this modal is a little awkwardly sized on an iphone, i think it would be better if it was wider